care/utils/assetintegration/onvif.py

```diff
@@ -55,3 +58,42 @@ def handle_action(self, action):
             return self.api_post(self.get_url("relativeMove"), request_body)
 
         raise ValidationError({"action": "invalid action type"})
+
+    def validate_action(self, action):
```

Comment:
instead of returning a boolean it would be better to raise a proper validation error